### DISCUSSION TITLE
HTML report execution date

### DISCUSSION THREAD

**Discussion by edsonmedina** [Codeception/Codeception#6186] [Ideas]
Can we add a "Generated by codeception in xxxx-xx-xx xx:xx:xx" footer to the HTML report?

It could be useful (for me at least) and I guess it wouldn't hurt.

Mind if I do a pull request with the changes?

**Comment by raistlin** [comment]
Please, create a pull request.

**Comment by nevercodealone** [comment]
Hi this would be very good

+1
